CMakeLists.txt

```
@@ -37,6 +37,8 @@ option(ICEBERG_BUILD_STATIC "Build static library" ON)
 option(ICEBERG_BUILD_SHARED "Build shared library" OFF)
 option(ICEBERG_BUILD_TESTS "Build tests" ON)
 option(ICEBERG_BUILD_BUNDLE "Build the battery included library" ON)
+option(ICEBERG_BUILD_ENABLE_ASAN "Enable Address Sanitizer" OFF)
+option(ICEBERG_BUILD_ENABLE_UBSAN "Enable Undefined Behavior sanitizer" OFF)
```

```suggestion
option(ICEBERG_ENABLE_ASAN "Enable Address Sanitizer" OFF)
option(ICEBERG_ENABLE_UBSAN "Enable Undefined Behavior sanitizer" OFF)
```

Let's make it shorter